tox.ini

```diff
@@ -139,7 +139,17 @@ deps =
   httpretty
 
 commands_pre =
-  python scripts/eachdist.py install --editable --with-test-deps
+  python -m pip install -e {toxinidir}/opentelemetry-api[test]
+  python -m pip install -e {toxinidir}/opentelemetry-sdk[test]
+  python -m pip install -e {toxinidir}/opentelemetry-instrumentation[test]
+  python -m pip install -e {toxinidir}/opentelemetry-proto[test]
+  python -m pip install -e {toxinidir}/tests/util[test]
+  python -m pip install -e {toxinidir}/instrumentation/opentelemetry-instrumentation-opentracing-shim[test]
+  python -m pip install -e {toxinidir}/exporter/opentelemetry-exporter-jaeger[test]
+  python -m pip install -e {toxinidir}/exporter/opentelemetry-exporter-opencensus[test]
+  python -m pip install -e {toxinidir}/exporter/opentelemetry-exporter-otlp[test]
+  python -m pip install -e {toxinidir}/exporter/opentelemetry-exporter-prometheus[test]
+  python -m pip install -e {toxinidir}/exporter/opentelemetry-exporter-zipkin[test]
```

Comment:
Sorry I've been heads down on the performance tests. Thanks for adding this change, although I find it weird that we need this (even if pip did upgrade) because isn't this what `eachdist.ini` `sortfirst` is supposed to be for?

https://github.com/open-telemetry/opentelemetry-python/blob/8d195e6a5a3c43a035f068dd7937ad47b32cd8e4/eachdist.ini#L7-L15

Reply:
The issue is more related to the fact that `eachdist` puts them all into one `pip install` statement, which apparently is buggy from the [recent release](https://github.com/pypa/pip/issues/8785).